Plugins/Player/NWScript/nwnx_player.nss

```diff
@@ -1002,3 +1020,33 @@ void NWNX_Player_UpdateWind(object oPlayer, vector vDirection, float fMagnitude,
     NWNX_PushArgumentObject(oPlayer);
     NWNX_CallFunction(NWNX_Player, sFunc);
 }
+
+void NWNX_Player_UpdateSkyBox(object oPlayer, int nSkyBox)
+{
+    string sFunc = "UpdateSkyBox";
+
+    NWNX_PushArgumentInt(nSkyBox);
+    NWNX_PushArgumentObject(oPlayer);
+    NWNX_CallFunction(NWNX_Player, sFunc);
+}
+
+
+void NWNX_Player_UpdateFogColor(object oPlayer, int nSunFogColor, int nMoonFogColor)
+{
+    string sFunc = "UpdateFogColor";
+
+    NWNX_PushArgumentInt(nMoonFogColor);
+    NWNX_PushArgumentInt(nSunFogColor);
+    NWNX_PushArgumentObject(oPlayer);
+    NWNX_CallFunction(NWNX_Player, sFunc);
+}
+
+void NWNX_Player_UpdateFogAmount(object oPlayer, int nSunFogAmount, int nMoonFogAmount)
+{
+    string sFunc = "UpdateFogAmount";
+
+    NWNX_PushArgumentInt(nMoonFogAmount);
+    NWNX_PushArgumentInt(nSunFogAmount);
+    NWNX_PushArgumentObject(oPlayer);
+    NWNX_CallFunction(NWNX_Player, sFunc);
+}
```

Comment:
Please include a newline at the end of every file, otherwise some tools choke and break